src/main/bin/ccm-initialise

```diff
@@ -128,9 +127,9 @@ MakeCacheRoot($this_app, $cache_root, $dopts);
 
 # make global lock file (mask and GID take care of the permissions)
 $this_app->verbose("Creating lockfile");
-open TMP, ">$cache_root/global.lock";
-print TMP "no\n";
-close TMP;
+open my $TMP, ">$cache_root/global.lock";
```

Comment:
@ned21 so you want to fail on all policies of severity 5? open an issue for that in maven-tools, i'll try to see how this can be done easily. but i'd rather have that done after new maven-tools release. i want to set a baseline in all repos with prs like this first, and then refine the thresholds.
even with the current small subset. it's already going to be a lot of work for all components.